web-app/src/components/common/Errorbox/styled.js

```diff
@@ -1,18 +1,25 @@
 import styled from 'styled-components';
 
 export const ErrorContainer = styled.div`
+<<<<<<< HEAD
```

Comment:
:(

Reply:
This component's merge conflict will be the end of me.